documentation/course/e2/README.md

```diff
@@ -1436,7 +1437,112 @@ Model danych serwisu jest na tyle prosty, a jednocześnie serwis tak uniwersalni
+
+![Diagram bazodanowy Clabbert](./images/database-diagram-clabbert.drawio.svg)
+
+<table>
+  <tr>
+    <th colspan="3">Indeksy</th>
+  </tr>
+  <tr>
+    <th>Kolumna</th>
+    <th>Typ</th>
+    <th>Opis</th>
+  </tr>
+  <tr>
+    <td><code>time_limited_offer.ticket_offer_id</code</td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>single_fare_offer.ticket_offer_id</code</td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>long_term_offer.ticket_offer_id</code</td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>ticket_offer.id</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>ticket.id</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>ticket.passenger_id</code></td>
+    <td>b-tree</td>
+    <td>Indeks wspierający wyszukiwanie biletów pasażera.</td>
+  </tr>
+  <tr>
+    <td><code>validation.id</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+  <tr>
+    <td><code>validation.ticket_id</code></td>
+    <td>b-tree (unikalny)</td>
+    <td>Indeks tworzony automatycznie przez bazę danych.</td>
+  </tr>
+    
+  <tr>
+    <th colspan="3">Ograniczenia</th>
+  </tr>
+  <tr>
+    <td colspan="3"><code>time_limited_offer.duration > 0</code></td>
```

Comment:
**question:** Can intervals be compared to unit-less numbers? (I don't know, but would guess that no)